packages/kit/src/exports/vite/dev/default_environment.js

```diff
@@ -0,0 +1,27 @@
+import { createNodeDevEnvironment } from 'vite';
+
+/**
+ * A default Node environment to pass to kit.environments.ssr in the Svelte config. This could eventually be used as the fallback for this option.
+ * @returns {(environment_name: string) => import('vite').Plugin[]}
+ */
+export function node() {
+	// @ts-ignore
```

Comment:
can we remove the `ts-ignore` or add a comment saying why the `ts-ignore` is required?